opencga-analysis/src/test/java/org/opencb/opencga/analysis/family/FamilyAnalysisTest.java

```diff
@@ -0,0 +1,220 @@
+package org.opencb.opencga.analysis.family;
+
+import org.junit.*;
+import org.junit.rules.ExpectedException;
+import org.opencb.biodata.models.clinical.Disorder;
+import org.opencb.biodata.models.clinical.Phenotype;
+import org.opencb.biodata.models.core.SexOntologyTermAnnotation;
+import org.opencb.biodata.models.pedigree.IndividualProperty;
+import org.opencb.commons.datastore.core.DataResult;
+import org.opencb.commons.datastore.core.ObjectMap;
+import org.opencb.commons.datastore.core.QueryOptions;
+import org.opencb.commons.test.GenericTest;
+import org.opencb.opencga.TestParamConstants;
+import org.opencb.opencga.analysis.tools.ToolRunner;
+import org.opencb.opencga.analysis.variant.OpenCGATestExternalResource;
+import org.opencb.opencga.analysis.variant.manager.VariantStorageManager;
+import org.opencb.opencga.catalog.exceptions.CatalogException;
+import org.opencb.opencga.catalog.managers.CatalogManager;
+import org.opencb.opencga.catalog.managers.FamilyManager;
+import org.opencb.opencga.catalog.utils.PedigreeGraphUtils;
+import org.opencb.opencga.core.api.ParamConstants;
+import org.opencb.opencga.core.exceptions.ToolException;
+import org.opencb.opencga.core.models.family.*;
+import org.opencb.opencga.core.models.individual.Individual;
+import org.opencb.opencga.core.models.individual.IndividualUpdateParams;
+import org.opencb.opencga.core.models.sample.Sample;
+import org.opencb.opencga.core.models.sample.SampleReferenceParam;
+import org.opencb.opencga.core.models.user.Account;
+import org.opencb.opencga.core.response.OpenCGAResult;
+import org.opencb.opencga.storage.core.StorageEngineFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class FamilyAnalysisTest extends GenericTest {
```

Comment:
Reminder : We'll need to add the `@Category` to this test once the categories are merged